label.rb

```diff
@@ -0,0 +1,16 @@
+class Label
+  attr_accessor :title, :color, :item
```

Comment:
Good work implementing the add label method, but this is maybe a little confusing. Let me explain how this should work in the application

- When you add a game, the application should ask you for the name of the label
- Once the game is added, you list the games. It should print the name of the games with the name on the label. Something like `World of Warcraft and the name of the label`.